modules/rpc-util.js

```diff
@@ -0,0 +1,40 @@
+const request = require('request');
+const Logger = require('./logger');
+const { Helper } = require('./helper');
+
+class RpcRequest {
+
+    static fetchBalance(provider, accountHash) {
+        var URL = `${provider}/chains/main/blocks/head/context/contracts/${accountHash}/balance`;
+        request(URL, function (error, response, body) {
+            if(error) {
+                Logger.error(`${error}`);
+            }
+            else {
+                try {
+                    var balance = JSON.parse(body);
+                    Logger.info(Helper.formatTez(balance));
+                }
+                catch(error) {
+                    Logger.error(`${error}`);
+                }
+            }
+        });
+    }
+
+    static checkNodeStatus() {
+        return new Promise(function(resolve, reject) {
+            request('http://localhost:18731/chains/main/blocks/head/protocols', function (error, response, body) {
```

Comment:
You can make this helper function more configurable by passing node address. Do not hardcode this kind of values in code.